Utilities/bin_patch.cpp

```diff
@@ -607,14 +596,14 @@ static std::basic_string<u32> apply_modification(const patch_engine::patch_info&
 		}
 		}
 
+		// Possibly an executable instruction
```

Comment:
Instead of aligning them down, seems better to not add unaligned patches to resval at all.